atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SnapshotTransaction.java

```diff
@@ -200,34 +196,34 @@
     /**
      * @param immutableTimestamp If we find a row written before the immutableTimestamp we don't need to
      *                           grab a read lock for it because we know that no writers exist.
-     * @param tokensValidForCommit These tokens need to be valid with {@link #lockService} for this transaction
+     * @param tokensValidForCommit These tokens need to be valid with {@link #timelockService} for this transaction
      *                             to commit.  If these locks have expired then the commit will fail.
```

Comment:
The Javadoc here seems out of date - I think it should read that the `AdvisoryLockPreCommitCheck` must pass, else the commit will fail? (and that check usually entails tokens valid with the plain `lockService`)